### ISSUE TITLE
Is HTML a programming language?

### ISSUE THREAD

**Issue by bofeiw** [shamsmosowi/git-bot#31]

**Comment by shamsmosowi** [comment]
No, HTML is not a programming language. HTML stands for HyperText Markup Language and is used to create the structure of a webpage. HTML is a markup language and does not contain any logic like a programming language does. It is used to define the structure of a page, such as headings, paragraphs, images, links, etc. HTML is not used to create dynamic content or to write code.
